src/pages/Checkout/CheckoutBetaPage.vue

```diff
@@ -203,7 +220,7 @@ export default {
 			currentStep: 'basket',
 			loans: [],
 			donations: [],
-			kivaCards: [],
+			kivaCards: () => [],
```

Comment:
@ksekhri Can you please set this back to just [] for now, just in case it affects how the pre-added kiva cards are handled... We'll experiment with this later...